### PR TITLE
[CP-2571] You can save Mudita Center into index.html file by clicking Alt + Left Click on Menu items

### PR DESCRIPTION
**Pull request by dkarski**
JIRA Reference: [CP-2571]

### :memo: Description ️

-

### :muscle: Checklist before requesting a review - nice to have

- getState function in async thunk actions is correctly typed
- redux selectors are used in components / prop drilling is reduce

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-2571]: https://appnroll.atlassian.net/browse/CP-2571?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/core/core/components/alt-link-download-preventer.component.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      const parentAnchor = (event.target as HTMLElement).closest("a")
+
+      if (!parentAnchor) {
+        return
+      }
+
+      event.preventDefault()
+      const href = parentAnchor.getAttribute("href")?.replace("#", "")
+
+      if (href && isValidPath(href)) {
```
Comment (lkowalczyk87) [inline libs/core/core/components/alt-link-download-preventer.component.tsx:29]: z.string().url().safeParse(val)

File: libs/core/core/components/use-alt-link-download-preventer-effect.hook.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/**
```
Comment (lkowalczyk87) [inline libs/core/core/components/use-alt-link-download-preventer-effect.hook.tsx:1]: Please move it to "shared"

File: libs/core/core/components/use-alt-link-download-preventer-effect.hook.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { useCallback, useEffect } from "react"
+
+const useAltLinkDownloadPreventerEffect = () => {
+  const handleClick = useCallback((event: MouseEvent) => {
+    if (event.altKey && !event.target) {
```
Comment (lkowalczyk87) [inline libs/core/core/components/use-alt-link-download-preventer-effect.hook.tsx:10]: Looks suspicious. If `altKey === false` and parent id anchor than prevent default.

File: libs/core/core/components/use-alt-link-download-preventer-effect.hook.tsx (hunk 2 above)
Comment (dkarski) [reply libs/core/core/components/use-alt-link-download-preventer-effect.hook.tsx:1]: Let's discuss on sync.

File: libs/core/core/components/use-alt-link-download-preventer-effect.hook.tsx (hunk 3 above)
Comment (dkarski) [reply libs/core/core/components/use-alt-link-download-preventer-effect.hook.tsx:10]: Let's discuss on sync.

**Review by lkowalczyk87 — APPROVED** [review]

**Pull request merged**